sonar-db/src/main/java/org/sonar/db/component/ResourceDao.java

```diff
@@ -136,11 +141,25 @@ public ResourceDto getRootProjectByComponentKey(String componentKey) {
   private ResourceDto getParentModuleByComponentId(Long componentId, DbSession session) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private method 'getParentModuleByComponentId' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)